library/node

```diff
@@ -53,6 +53,26 @@ Architectures: arm32v7, arm64v8, amd64, i386, ppc64le, s390x
 GitCommit: 72dd945d29dee5afa73956ebc971bf3a472442f7
 Directory: 6/stretch
 
+Tags: 11.0.0-jessie, 11.0-jessie, 11.0.0, 11.0
+Architectures: arm32v7, arm64v8, amd64, i386, ppc64le, s390x
+GitCommit: e007863a848ca34221bb24549293567e2336445b
+Directory: 11/jessie
```

Comment:
To be totally frank, why add `11-jessie` at all?  Why not simply let Debian Jessie support die with Node.js 11?

IMO, 11.0, etc should point to stretch instead, especially since this is a brand new release (previously unsupported completely).

Shouldn't this also have at least `11` as an additional alias?

Reply:
@tianon maybe need your feedback here: https://github.com/nodejs/docker-node/pull/863, thanks.

Reply:
I think the feedback I provided in my original post there was pretty comprehensive -- the bottom line is that the way the Node.js Docker repo is tracking "supported" architectures is extremely error-prone without any good reason that I can see.

To @chorrell's specific follow-up question there (which from what I can tell shouldn't have been blocking the merge), IMO, y'all should've switched `node:*-slim` over to `stretch` ages ago, but yes, you'll need to update the `architectures` files again in the opposite direction if you update `slim` to `stretch`.